AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -34,6 +34,8 @@ turkuazgazetesi.net,yenialanya.com,mansetaydin.com,seskocaeli.com,kocaelibarisga
 !
 ! SECTION: Subscriptions - Regular rules
 !
+icij.org##.modal
+icij.org##.modal-backdrop
```

Comment:
I would use this rule `icij.org##body > div[no-headings][style^="position:"]` instead of those ones.